bundles/org.openhab.core.automation/src/main/resources/OH-INF/automation/moduletypes/EphemerisConditions.json

```diff
@@ -10,7 +10,7 @@
 					"type": "INTEGER",
 					"label": "Offset",
 					"description": "Today +/- offset days (+1 = tomorrow, -1 = yesterday).",
-					"default": 0,
+					"defaultValue": 0,
```

Comment:
Imho, the JSON representation of a config description param should be identical to the XML representation where this attribute is called `default`. The only reason why it is called `defaultValue` in the DTO is that `default` is a reserved keyword in Java and thus cannot be used there.
The better fix would therefore imho be to adjust the JSON parsing.

Reply:
We can add the `@SerializedName` annotation in the ConfigDescriptionParameterDTO` for this field. e.g.

```java
@SerializedName(value="default", alternate="defaultValue")
```

The first one ("value") will be used as a default for serialization and deserialization, while the second ("defaultValue") will be used to indicate an alternate name of the JSON field that we want to map. But in this way we will introduce a regression as it will be used for exposing configuration description via REST interface too. Which name should be used for exposing the DTO?

Reply:
@ghys What's your opinion on the breaking change? Is this something that can be easily adjusted in the Main UI?
It seems to me as half of our config descriptions might not work since we do not use the same attribute name in all places. As it is an optional one, I'd actually hope that the change won't really cause issues in the UI, it might just not recognize the "other half" config descriptions, which currently use `defaultValue` in their JSON.

Reply:
I count only 4 (relevant) occurrences of `defaultValue` in the UI's code, 2 here:
https://github.com/openhab/openhab-webui/blob/c71db73a6163026f383d352ace8362bc41ccd426/bundles/org.openhab.ui/web/src/components/config/config-sheet.vue#L88
and 2 related to https://github.com/openhab/openhab-webui/pull/1049.
So overall it's feasible but might break a few widgets that make use of the feature in the above PR (which is relatively recent so no big impact anyways).

Reply:
Thanks for checking - should be a rather safe change then.